ansible/container-infra-upgrade.yml

```diff
@@ -0,0 +1,73 @@
+- hosts:
+  - minion
+  vars:
+    minion_services:
+    - name: kubernetes-kubelet
+      service: kubelet
+    - name: kubernetes-proxy
+      service: kube-proxy
+  tasks:
+  - name: pull images
+    # command: "atomic pull --storage ostree docker.io/openstackmagnum/{{ item.name }}:{{ item.version | default(version) }}"
```

Comment:
nit: Looks like this can be removed?

Reply:
I've left it here for the time being as a note for what atomic command this is equivalent to.